OAuth/ResourceOwner/Auth0ResourceOwner.php

```diff
@@ -34,19 +34,35 @@ class Auth0ResourceOwner extends GenericOAuth2ResourceOwner
         'profilepicture' => 'picture',
     );
 
+    protected function getRequestHeaders($extends = array()) {
+
+        return array_merge($extends, array(
+            'User-Agent' => 'PHP/' . phpversion(),
+            'Auth0-Client' => 'PHP/HWIOAuthBundle',
```

Comment:
Yes it is possible but I don't see the `use-case`; 
can you be more specific please ?